docs/pages/api-reference/liveblocks-client.mdx

```diff
@@ -53,6 +53,10 @@ Your async function should return either of the following values:
 Any other error will be treated as an unexpected error, after which the client
 will retry making the request until it gets either (1) or (2).
 
+Note that when using Notifications, the param `room` can be undefined. This
+means that the client requests a token that grants access to multiple rooms
+instead of a specific room.
```

Comment:
```suggestion
Note that when using Notifications, the parameter `room` can be undefined. This implies that the client is requesting a token that grants access to multiple rooms, rather than a specific room.
```